ui/raidboss/data/07-dt/trial/valigarmanda.ts

```diff
@@ -556,6 +556,41 @@ const triggerSet: TriggerSet<Data> = {
         'Tulidisaster': '图拉尔灾祸',
       },
     },
+    {
+      'locale': 'ko',
+      'replaceSync': {
+        'Arcane Sphere': '입체마법진',
+        'Flame-kissed Beacon': '불의 상징',
+        'Valigarmanda': '발리가르만다',
+      },
+      'replaceText': {
+        '\\(AoEs\\)': '(장판)',
+        '\\(ice phase\\)': '(얼음 페이즈)',
+        '\\(knockback\\)': '(넉백)',
+        '\\(storm phase\\)': '(번개 페이즈)',
+        '\\(tower\\)': '(기둥)',
+        '\\(storm/ice phase\\?\\)': '(번개/얼음 페이즈?)',
```

Comment:
(not directly related to this PR)

Other languages seem to be missing this replacement. Is it not showing in the "find missing timeline translations" tool, or was it just not translated?

Reply:
https://github.com/OverlayPlugin/cactbot/blob/3ebdd7a9612c9de75f40ba20e87f56c585f3853c/ui/raidboss/data/07-dt/trial/valigarmanda.txt#L101

Parenthesized phrases that are not present in .ts files, but only in .txt files, are not checked by the find-translations tool, and do not seem to be treated as missing translations.